CHANGELOG.md

```diff
@@ -7,19 +7,22 @@ Note that the displayed date is in the format `dd-mm-yyyy`
 [Older changelogs](#older-changelogs)
 
 ## [v0.2.0]
+
```

Comment:
Bro, even without these changes it will work

Reply:
This is optional

Reply:
Yes it will work. But it sure does violate the rules.